tests/test_raw.py

```diff
@@ -38,77 +62,65 @@ def test_delete_databases(self):
-#         assert isinstance(tables.to_pandas(), pd.DataFrame)
-#
-#     def test_delete_tables(self):
-#         response = raw.delete_tables(database_name=DB_NAME, table_names=[TABLE_NAME])
-#         assert response == {}
-#         with pytest.raises(APIError) as e:
-#             raw.delete_tables(DB_NAME, [TABLE_NAME])
-#             print(e.code)
-#             assert 0
-
-#
-#
-# @pytest.fixture(scope='module')
-# def rows():
-#     return raw.get_rows(database_name=DB_NAME, table_name=TABLE_NAME, limit=1)
-#
-#
-# def test_create_rows():
-#     response = raw.create_rows(DB_NAME, TABLE_NAME, rows=[RawRowDTO(key=ROW_KEY, columns=ROW_COLUMNS)])
-#     assert response == {}
-#
-#
-# # Test getting rows and their resulting output formats
-# def test_databases_output_formats(databases):
-#
-#
-
-#
-#
-
-#
-#
-
-#
-#
-# def test_rows_response_length(rows):
-#
-#
-#
-#
-# def test_rows_object_output_formats(rows):
-#     assert isinstance(rows, RawObject)
-#     assert isinstance(rows.to_json(), list)
-#     assert isinstance(rows.to_ndarray(), np.ndarray)
-#     assert isinstance(rows.to_pandas(), pd.DataFrame)
+
+            
+class TestTables:
+    @pytest.fixture(autouse=True, scope='class')
+    def create_database(self):
+        raw.create_databases([DB_NAME])
+        yield
+        raw.delete_databases([DB_NAME], recursive=True)
+
+    @pytest.fixture(scope='class')
+    def tables(self):
+        yield raw.get_tables(DB_NAME)
+
+    def test_create_tables(self):
+        response = raw.create_tables(DB_NAME, [TABLE_NAME])
+        # assert isinstance(response, RawObject)
+        assert response.to_json()[0]['tableName'] == TABLE_NAME
+
+    def test_tables_response_length(self, tables):
+        assert len(tables.to_json()) > 0
+
+    def test_tables_object_output_formats(self, tables):
+        assert isinstance(tables, RawResponse)
+        assert isinstance(tables.to_json(), list)
+        assert isinstance(tables.to_ndarray(), np.ndarray)
+        assert isinstance(tables.to_pandas(), pd.DataFrame)
+
+    def test_delete_tables(self):
+        response = raw.delete_tables(database_name=DB_NAME, table_names=[TABLE_NAME])
+        assert response == {}
+        with pytest.raises(APIError) as e:
+            raw.delete_tables(DB_NAME, [TABLE_NAME])
+            # assert re.match("{'code': 404, 'message': 'Did not find any dbs with the given names'}", str(e.value))
+            # assert re.match("{'code': 404, 'message': 'No tables named test_table'}")
+
+
+class TestRows:
+    @pytest.fixture(autouse=True, scope='class')
+    def create_database(self):
+        raw.create_databases([DB_NAME]).to_json()
+        raw.create_tables(DB_NAME, [TABLE_NAME]).to_json()
+        yield
+        raw.delete_databases([DB_NAME], recursive=True)
+
+    def test_create_rows(self):
+        response = raw.create_rows(DB_NAME, TABLE_NAME, rows=[RawRowDTO(key=ROW_KEY, columns=ROW_COLUMNS)])
```

Comment:
Create rows response is equals to empty array?

Reply:
yes